go/oasis-test-runner/scenario/e2e/runtime/sentry.go

```diff
@@ -189,9 +176,7 @@ func (s *sentryImpl) Run(ctx context.Context, childEnv *env.Env) error { // noli
 		return fmt.Errorf("failed to get status for compute1: %w", err)
 	}
 	compute1ServerPublicKeys := make(map[signature.PublicKey]bool)
```

Comment:
This variable is never used :shrug:

Reply:
Even though this is resolved, it still does not seem to be used anywhere? In that case we should just remove it.